### DISCUSSION TITLE
Analytics API is not working

### DISCUSSION THREAD

**Discussion by chienhsiang-hung** [vercel/vercel#8284] [Help] [answered]
I have a project set up Analytics API like the examples in https://vercel.com/docs/concepts/analytics/api

My repo:
in root [vitals.js](https://github.com/chienhsiang-hung/kaggle-profile-summary-card/blob/main/vitals.js)

in [src](https://github.com/chienhsiang-hung/kaggle-profile-summary-card/tree/main/src:)
`reportWebVitals.js`, `vitals.js`

I've tried many times but it just won't work. If you could kindly help me out with this would be so appreciated.

**Comment by amyegan** [comment, marked as answer]
Hi @chienhsiang-hung. Thanks for reaching out! It looks like you added the example functions but are not calling them from anywhere in your app. You'll need to add a call to the function like in this [React example](https://github.com/vercel/vercel/blob/4e4f5f28a24f7fc8b29e669c07a683520daa5331/examples/create-react-app/src/index.js) to post the information.

The important part is the your post the data to `/v1/vitals` with the appropriate parameters as listed in [Measure web vitals](https://vercel.com/docs/concepts/analytics/api#endpoints/vitals/measure-web-vitals). That can be done in whatever way makes sense for you. It doesn't need to look exactly like the code example at the bottom of the Analytics API page.
